Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -19,6 +19,7 @@
 use Wirecard\Oxid\Core\PostProcessingHelper;
 use Wirecard\Oxid\Core\TransactionHandler;
 use Wirecard\Oxid\Model\Transaction;
+use Wirecard\Oxid\Model\PaymentMethod\PaymentMethod;
```

Comment:
Why was this added? I don't think it's used.

Reply:
Removed 👍